src/Assets/citizenshipFilterFormControlLabels.tsx

```diff
@@ -11,52 +10,36 @@ export type CitizenLabels =
   | 'gc_under19_pregnant_no5';
 
 const citizenshipFilterFormControlLabels = {
-  citizen: (
-    <FormattedMessage
-      id="citizenshipFCtrlLabel-citizen"
-      defaultMessage="Show benefits that require citizenship or lawful U.S. presence"
-    />
-  ),
   non_citizen: (
     <FormattedMessage
       id="citizenshipFCtrlLabel-non_citizen"
-      defaultMessage="Show benefits that do not require citizenship or lawful U.S. presence"
-    />
-  ),
-  green_card: (
-    <FormattedMessage
-      id="citizenshipFCtrlLabel-green_card"
-      defaultMessage="Show benefits available to green card holders"
-    />
-  ),
-  refugee: (
-    <FormattedMessage
-      id="citizenshipFCtrlLabel-refugee"
-      defaultMessage="Show benefits available to refugees or asylees (special rules or waiting periods may apply)"
-    />
-  ),
-  gc_5plus: (
-    <FormattedMessage
-      id="citizenshipFCtrlLabel-gc_5plus"
-      defaultMessage="Show benefits available to green card holders ONLY after 5+ years in the U.S."
+      defaultMessage="Individuals without citizenship or lawful U.S. presence"
     />
   ),
+  green_card: <FormattedMessage id="citizenshipFCtrlLabel-green_card" defaultMessage="Green card holders" />,
```

Comment:
This will require adding the requested text changes in our new translations UI via api/translations/admin